app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -143,6 +148,27 @@ class Scene extends React.Component {
     'count',
   ]);
 
+  addToOverlay = body => {
+    if (body && body.render.sprite.texture) {
+      const ctx = this.overlay.current.getContext('2d');
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Optimizing_canvas#Turn_off_transparency

Reply:
Doing this seems to destroy everything on the first canvas. Note the lack of counts at the top, and the blocks that are falling are not rendered either
![Screen Shot 2019-06-24 at 2 55 50 PM](https://user-images.githubusercontent.com/48894331/60044366-30d17080-9690-11e9-8059-8009d8ebeec1.png)